src/components/core/wizard/wizard.tsx

```diff
@@ -0,0 +1,401 @@
+import CloseIconLight from '@assets/svg/modal/ic-close-light.svg';
+import BackArrowIconLight from '@assets/svg/wizard/ic-back-arrow-light.svg';
+import BackArrowIconDark from '@assets/svg/wizard/ic-back-arrow-dark.svg';
+import Button from '@core/button/button';
+import Text from '@core/text/text';
+import React from 'react';
+import { styled } from 'Styles/stitches.config';
+import StepNavigation from './step-navigation';
+
+const DarkBackgroundContainer = styled('div', {
+    position: 'absolute',
+    inset: '0',
+    background: 'rgba(0, 0, 0, 0.72)',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+
+    variants: {
+        visible: {
+            false: {
+                display: 'unset',
+                background: 'unset',
+            },
+        },
+    },
+});
+
+const WizardContainer = styled('div', {
+    position: 'relative',
+    width: '1040px',
+    height: '640px',
+    backgroundColor: '$white',
+    borderRadius: '16px',
+    display: 'flex',
+    overflow: 'hidden',
+    boxShadow: '0px 0px 24px rgba(0, 0, 0, 0.08), 0px 24px 24px rgba(0, 0, 0, 0.08)',
+
+    variants: {
+        dark: {
+            true: {
+                backgroundColor: '#0E0E0E',
+            },
+        },
+    },
+});
+
+const LeftPanel = styled('div', {
+    width: '256px',
+    height: '640px',
+    backgroundColor: '#F2F3F4',
+    boxSizing: 'border-box',
+    padding: '48px 24px',
+
+    variants: {
+        dark: {
+            true: {
+                backgroundColor: '#151717',
+            },
+        },
+    },
+});
+
+const MainTitleContainer = styled('div', {
+    display: 'flex',
+    gap: '10px',
+});
+
+const WizardBody = styled('div', {
+    width: '784px',
+    height: '640px',
+});
+
+const ContentContainer = styled('div', {
+    width: '784px',
+    height: '568px',
+    position: 'relative',
+    boxSizing: 'border-box',
+    padding: '48px 24px 24px',
+});
+
+const FixedWidthContainer = styled('div', {
+    width: '472px',
+    height: '568px',
+    boxSizing: 'border-box',
+    padding: '48px 24px 24px',
+    position: 'absolute',
+    top: '0',
+    left: '0',
+    bottom: '0',
+});
+
+const RightPanel = styled('div', {
+    width: '312px',
+    height: '568px',
+    boxSizing: 'border-box',
+    padding: '48px 24px 24px',
+    position: 'absolute',
+    top: '0',
+    right: '0',
+    bottom: '0',
+    borderLeft: '2px solid #F2F3F4',
+
+    variants: {
+        dark: {
+            true: {
+                borderLeft: '2px solid #323738',
+            },
+        },
+    },
+});
+
+const RightPanelBlock = styled('div', {
+    boxSizing: 'border-box',
+
+    variants: {
+        upper: {
+            true: {
+                minHeight: '272px',
+                paddingBottom: '24px',
+                borderBottom: '1px solid #D6DADB',
+            },
+        },
+        lower: {
+            true: {
+                paddingTop: '24px',
+            },
```

Comment:
Can `upper` & `lower` be used together? If not, then we can group them under one variant.